client/docx_generator/README.md

```diff
@@ -0,0 +1,144 @@
+# King-Phisher Plugin
+This plugin will take an docx attachment and embed King Phisher Jinja variables into it when sending the email to a target.
+With this plugin you can check how many people opened the Word attachment and choose the option to activate the macro or their macro execution was already active.
+
+# Prerequisites
+note: The client and server must not be running.
+
+you can run below command in install king-phisher directory for example, suppose you installed it at /opt/king-phisher then:
+
+    cd /opt/king-phisher
+
+    pipenv install docxtpl
+    
+# Local install
+1- Locate the docx_generator folder in the following path:
```

Comment:
Proper markdown would format this numbered list as `1.` and not `1-`. Each of the numbers will need to be updated to reflect this formatting change.